src/main/js/components/LogIn.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div class="login">
+    <input name="email" type="email" placeholder="Email Address">
+    <input name="password" type="password" placeholder="Password">
+    {{ incorrect_user_info }}
+    <input type="button" v-on:click="login" value="Log In">
+  </div>
+</template>
+
+<script>
+import axios from "axios";
+
+export default {
+  data() {
+    return {
+      user: { email: null, password: null },
+      incorrect_user_info: null
+    }
+  },
+  methods: {
+    login: function () {
+      // ここでAPIたたく
+      let email = this.$data.user.email;
+      let password = this.$data.user.password;
+
+      // TODO: ここでAPIたたく
+      let isCorrectUserInfo = "あとで実装"
+      this.incorrect_user_info = isCorrectUserInfo ? "" : "Login information is incorrect."
+    }
+  }
```

Comment:
ここにcsrfトークンをgetするメソッドを記述するとレンダリングした後にフックされる
```suggestion
  }
  mounted(){
    csrf: function(){
      const csrf = axios.get("/prelogin");
      return csrf;
  }
```

Reply:
それっぽい修正した